.mise/tasks/platform/pricemodel/deploy/all

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+
+# mise description="deploy price model to all environments"
+# mise depends=["platform:pricemodel:deploy:development", "platform:pricemodel:deploy:paper", "platform:pricemodel:deploy:live"]
```

Comment:
Should `paper` and `live` be removed?

Reply:
I think we should keep them in general
the namespaces are still there
I just took down the services/pods that were in there for now